docs/library/btree.rst

```diff
@@ -72,6 +72,18 @@ Example::
     # Don't forget to close the underlying stream!
     f.close()
 
+.. note::
+
+    Due to licensing restrictions ``btree`` is not commonly included in builds, e.g. PyBoard builds.
+    The licensing restriction is that when advertising products containing the BerkelyDB library then
+    the advertising must acknoledge that the BerkelyDB library (this is onerous to do in *all* advertising).
```

Comment:
`acknoledge that` -> `acknowledge` ? Although it might be better to just link to the file or document with the license because 'advertising' is a bit vague and the exact license text differs a bit from what is written here (see btree.h)